tests/tests/optionsTests.robot

```diff
@@ -114,4 +110,20 @@ Enable Debug Output functionality test
     Check If Log Console Contains    loaded
     Wait Until Element Is Visible    ${ReloadBadge}
 
-
+Verify Output log to a file option
+    [Documentation]    Verify Output log to a file option
+    [Tags]    options
+    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${file_4mb}[archive_name].zip
+    Click Add Files Button
+    Ubuntu Upload Test File    ${file_4mb}[path]
+    Click Load Button
+    Check If Log Console Contains    Opening "${file_4mb}[name]"
+    Validate Output Description    ${file_4mb}[archive_name]
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${LogsButton}
+    Click Element    ${OutputLogsSwitch}
+    Wait Until Element Is Visible    ${DownloadLogsButton}
+    Click Load Button
+    Wait Until Keyword Succeeds  5s  1s  Click Element  ${DownloadLogsButton}
+    Switch Window    new
+    Wait Until Element Is Visible    ${OpeningFileText}
```

Comment:
As a general comment I think this test could be a little bit shorter to make it more clear. Lines 119-121 could be removed, there is no need to load file twice in this test.

Reply:
done